Updated STM code for GR4

GSweet96 opened the pull request:
Updated STMFragment
Updated STMWaveformDigi
Updated Digi maker code

FNALbuild commented:
Hi @GSweet96,
You have proposed changes to files in these packages:
- RecoDataProducts
- DAQ
- STMReco

which require these tests: build.

@Mu2e/write, @Mu2e/fnalbuild-users have access to CI actions on [main](/Mu2e/Offline/tree/main).





:hourglass: The following tests have been triggered for e73dc7d2f39018b6ea563ff9b954d3ba9dd038b0: build  (Build queue is empty)


[About FNALbuild.](https://mu2ewiki.fnal.gov/wiki/GitHubWorkflow#GitHub_Pull_Request_Procedures_and_FNALbuild) [Code review on Mu2e/Offline.](https://mu2ewiki.fnal.gov/wiki/GitHubWorkflow#Code_Review)

FNALbuild commented:
:umbrella: The build is failing at e73dc7d2f39018b6ea563ff9b954d3ba9dd038b0.

```
scons: *** Multiple ways to build the same target were specified for: /scratch/workspace/GitHubPRTests/mu2e-offline-build-test/build/al9-prof-e28-p068/mu2e_trig_config/gen/trig_physMenuPSConfig_OnSpill.fcl  (from ['/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/data/physMenu.json', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/data/extrPosMenu.json', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigDigiInputsEpilog.fcl', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigFilters.fcl', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigProducers.fcl', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigSequences.fcl', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigServices.fcl'] and from ['/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/data/physMenu.json', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/data/physMenu.json', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/data/extrPosMenu.json', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigDigiInputsEpilog.fcl', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigFilters.fcl', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigProducers.fcl', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigSequences.fcl', '/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/mu2e_trig_config/core/trigServices.fcl'])
```


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| test with | :white_check_mark: | Command did not list any other PRs to include |
| merge | :white_check_mark: | Merged e73dc7d2f39018b6ea563ff9b954d3ba9dd038b0 at e2e7df15eb788888bbed7ac1c1635074e1c383f9 |
| build (prof) | :x: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/scons.log).  |
| ceSimReco | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/ceSimReco.log)  |
| g4test_03MT | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/g4test_03MT.log)  |
| transportOnly | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/transportOnly.log)  |
| POT | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/POT.log)  |
| g4study | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/g4study.log)  |
| cosmicSimReco | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/cosmicOffSpill.log)  |
| ceSteps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/ceSteps.log)  |
| ceDigi | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/ceDigi.log)  |
| muDauSteps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/muDauSteps.log)  |
| ceMix | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/ceMix.log)  |
| rootOverlaps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/g4surfaceCheck.log)  |
| trigger | :heavy_exclamation_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/trigger.log)  |
| check_cmake | :heavy_exclamation_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/check_cmake.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 17 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/fixme_todo.log) |
| clang-tidy | :wavy_dash: | [0 errors 0 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/clang-tidy.log) |
| whitespace check | :large_orange_diamond: | [found whitespace errors](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/artifact/whitespace_errs.log) |


N.B. These results were obtained from a build of this Pull Request at e73dc7d2f39018b6ea563ff9b954d3ba9dd038b0 after being merged into the base branch at e2e7df15eb788888bbed7ac1c1635074e1c383f9.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2423/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.

GSweet96 commented:
**RecoDataproducts/inc/STMWaveformDigi.hh :**

Added Detector ID, DTC clock (40MHz), ADC clock (75MHz).

- DetID() // Channel ID of detector ( 0 = HPGe, 1 = LaBr (not yet used) )
- DTCtime() // 40 MHz event/trigger time, derived from 200MHz DTC clock
- ADCtime() // 75 MHz event/trigger time, derived from 300MHz ADC clock
- peak_fitTime1() // Rising edge fit time of first peak (ns)
- peak_fitTime2() // Rising edge fit time of second peak (ns)
- peak_sep() // Separation of peaks 1&2 (ns)

**DAQ/src/STMWaveformDigisFromFragments_module.cc**

Added method to get above Digi parameters 
includes new fhicl parameters:

- width_guess // "Estimate of pulse width for fit (default = 150e-9)
- rise_time_guess // "Estimate of pulse rise time for fit (default = 6e-9)"
- threshold // "Threshold to define the peak (default = 20000)"
- samp_freq // "Sampling frequency of ADC (default = 300e6)"
- pulse_region "Number of ADC values to fit around pulse (default 51)";
- pulses_per_event // "Number of pulses per event to find Delta t (default = 2)"

**STMReco/src/STMAnalyzeDigis_module.cc**

New module for Offline processing of Digis including pulse fitting and making plots.

brownd1978 commented:
@FNALbuild run build test

FNALbuild commented:
:hourglass: The following tests have been triggered for 045d9e2ea35bd9f599924509a0bcd077a5475ed0: build  (Build queue is empty)

FNALbuild commented:
:umbrella: The build is failing at 045d9e2ea35bd9f599924509a0bcd077a5475ed0.

```
scons: *** [build/al9-prof-e28-p071/Offline/tmp/DAQ/src/STMWaveformDigisFromFragments_module.os] Error 1
scons: *** [build/al9-prof-e28-p071/Offline/tmp/STMMC/src/HPGeWaveformsFromGeantSim_module.os] Error 1
scons: *** [build/al9-prof-e28-p071/Offline/tmp/STMReco/src/STMAnalyzeDigis_module.os] Error 1
scons: *** [build/al9-prof-e28-p071/Offline/tmp/STMReco/src/STMZeroSuppression_module.os] Error 1
scons: *** [build/al9-prof-e28-p071/Offline/tmp/Sources/src/FromSTMTestBeamData_source.os] Error 1
```


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| test with | :white_check_mark: | Command did not list any other PRs to include |
| merge | :white_check_mark: | Merged 045d9e2ea35bd9f599924509a0bcd077a5475ed0 at 967897ed5eb5ebd67bc635cc0b6e503579409c11 |
| build (prof) | :x: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/scons.log).  |
| ceSimReco | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/ceSimReco.log)  |
| g4test_03MT | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/g4test_03MT.log)  |
| transportOnly | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/transportOnly.log)  |
| POT | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/POT.log)  |
| g4study | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/g4study.log)  |
| cosmicSimReco | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/cosmicOffSpill.log)  |
| ceSteps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/ceSteps.log)  |
| ceDigi | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/ceDigi.log)  |
| muDauSteps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/muDauSteps.log)  |
| ceMix | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/ceMix.log)  |
| rootOverlaps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/g4surfaceCheck.log)  |
| trigger | :heavy_exclamation_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/trigger.log)  |
| check_cmake | :heavy_exclamation_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/check_cmake.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 6 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/fixme_todo.log) |
| clang-tidy | :large_orange_diamond: | [6 errors 313 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/clang-tidy.log) |
| whitespace check | :large_orange_diamond: | [found whitespace errors](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/artifact/whitespace_errs.log) |


N.B. These results were obtained from a build of this Pull Request at 045d9e2ea35bd9f599924509a0bcd077a5475ed0 after being merged into the base branch at 967897ed5eb5ebd67bc635cc0b6e503579409c11.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2444/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.

gianipez commented:
@FNALbuild run build test

brownd1978 commented:
@FNALbuild run build test

FNALbuild commented:
:hourglass: The following tests have been triggered for 2cfbf697a15f17320ccc4d53afa5470bd2039688: build  (Build queue has 2 jobs)

FNALbuild commented:
:umbrella: The build is failing at 2cfbf697a15f17320ccc4d53afa5470bd2039688.

```
scons: *** [build/al9-prof-e28-p071/Offline/tmp/DAQ/src/STMWaveformDigisFromFragments_module.os] Error 1
scons: *** [build/al9-prof-e28-p071/Offline/tmp/STMMC/src/HPGeWaveformsFromGeantSim_module.os] Error 1
scons: *** [build/al9-prof-e28-p071/Offline/tmp/STMReco/src/STMAnalyzeDigis_module.os] Error 1
scons: *** [build/al9-prof-e28-p071/Offline/tmp/STMReco/src/STMZeroSuppression_module.os] Error 1
scons: *** [build/al9-prof-e28-p071/Offline/tmp/Sources/src/FromSTMTestBeamData_source.os] Error 1
```


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| test with | :white_check_mark: | Command did not list any other PRs to include |
| merge | :white_check_mark: | Merged 2cfbf697a15f17320ccc4d53afa5470bd2039688 at bf22af5f7ce8841dcfe8d36a426a3841f3814da5 |
| build (prof) | :x: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/scons.log).  |
| ceSimReco | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/ceSimReco.log)  |
| g4test_03MT | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/g4test_03MT.log)  |
| transportOnly | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/transportOnly.log)  |
| POT | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/POT.log)  |
| g4study | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/g4study.log)  |
| cosmicSimReco | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/cosmicOffSpill.log)  |
| ceSteps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/ceSteps.log)  |
| ceDigi | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/ceDigi.log)  |
| muDauSteps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/muDauSteps.log)  |
| ceMix | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/ceMix.log)  |
| rootOverlaps | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :wavy_dash: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/g4surfaceCheck.log)  |
| trigger | :heavy_exclamation_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/trigger.log)  |
| check_cmake | :heavy_exclamation_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/check_cmake.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 6 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/fixme_todo.log) |
| clang-tidy | :large_orange_diamond: | [6 errors 303 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/clang-tidy.log) |
| whitespace check | :large_orange_diamond: | [found whitespace errors](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/artifact/whitespace_errs.log) |


N.B. These results were obtained from a build of this Pull Request at 2cfbf697a15f17320ccc4d53afa5470bd2039688 after being merged into the base branch at bf22af5f7ce8841dcfe8d36a426a3841f3814da5.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2447/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.

gianipez commented:
@FNALbuild run build test

FNALbuild commented:
:hourglass: The following tests have been triggered for fbcdb42d3e57330f29e1622b54b17ac4597dfee6: build  (Build queue is empty)

FNALbuild commented:
:sunny: The build tests passed at fbcdb42d3e57330f29e1622b54b17ac4597dfee6.


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| test with | :white_check_mark: | Command did not list any other PRs to include |
| merge | :white_check_mark: | Merged fbcdb42d3e57330f29e1622b54b17ac4597dfee6 at 3167110c774264a232c50efc664d03b23ae70a93 |
| build (prof) | :white_check_mark: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/scons.log). Build time: 04 min 27 sec |
| ceSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/ceSimReco.log)  |
| g4test_03MT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/g4test_03MT.log)  |
| transportOnly | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/transportOnly.log)  |
| POT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/POT.log)  |
| g4study | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/g4study.log)  |
| cosmicSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/cosmicOffSpill.log)  |
| ceSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/ceSteps.log)  |
| ceDigi | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/ceDigi.log)  |
| muDauSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/muDauSteps.log)  |
| ceMix | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/ceMix.log)  |
| rootOverlaps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/g4surfaceCheck.log)  |
| trigger | :question: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/trigger.log) Return Code 1. |
| check_cmake | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/check_cmake.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 6 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/fixme_todo.log) |
| clang-tidy | :large_orange_diamond: | [6 errors 287 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/clang-tidy.log) |
| whitespace check | :large_orange_diamond: | [found whitespace errors](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/artifact/whitespace_errs.log) |


N.B. These results were obtained from a build of this Pull Request at fbcdb42d3e57330f29e1622b54b17ac4597dfee6 after being merged into the base branch at 3167110c774264a232c50efc664d03b23ae70a93.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//2451/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.
